docs/agents/sematext-agent/windows-installation.md

```diff
@@ -235,7 +257,7 @@ cd C:\Windows\System32\config\systemprofile\AppData\Local\sematext-agent
 notepad.exe .\logs\st-agent.log
 ```
 
-## How to change Infra App token or region
+## How to change App tokens or region
 
```

Comment:
@boratanrikulu can you enter the exact command to change Infra & Logs token here.